### DISCUSSION TITLE
Questions about 2 hardware

### DISCUSSION THREAD

**Discussion by szcsaba101** [helloSystem/hello#482] [Q&A]
Hello!
I have 2 questions. I have a Steinberg ur-22 sound card. Does this work on hello system?
The internet does not work with my current wifi card. If I bought the d-link-dwa-131-wifi-adapter, would it work?
Thanks!

**Comment by probonopd** [comment]
Hello.

I don't know about your WLAN adapter, but I have success with a $3 one from the big auction site:

![image](https://user-images.githubusercontent.com/2480569/219771072-b93ba364-d723-4e54-8b6a-52c0c00d1661.png)

The Steinberg UR22 **MK II** has a "Class Compliant" mode and should hence work (but since I don't have one I can't test this). Best way to find out whether yours works is to try it out with the helloSystem 0.8.1 Live ISO.

**Reply by szcsaba101** [reply]
Thanks a lot! l am glad!
My motherboard: ASROCK H570M Pro4.
l will try newest hello system just l gotta buy a new wifi card.

**Reply by probonopd** [reply]
In the meantime, you might also be able to use your Android phone, connected via USB, in tethering mode, for WLAN. helloSystem 0.8.1 has support for it.
